chapters/prg/python/python-install.md

```diff
@@ -334,30 +334,21 @@ guided procedures based on what OS you are using.  You can find the latest instr
 * MacOS. <https://conda.io/projects/conda/en/latest/user-guide/install/macos.html>
 * Linux. <https://conda.io/projects/conda/en/latest/user-guide/install/linux.html>
 
-### Installing Python via `conda`
```

Comment:
I moved this section down further, as the conda command needs to be added to the path first